rc/base/autowrap.kak

```diff
@@ -37,9 +39,9 @@ def -hidden autowrap-cursor %{ eval -save-regs '/"|^@m' %{
 } }
 
 def autowrap-enable -docstring "Wrap the lines in which characters are inserted" %{
-    hook -group autowrap window InsertChar [^\n] autowrap-cursor
+    hook -group autowrap buffer InsertChar [^\n] autowrap-cursor
```

Comment:
well, if we put the hook in window scope, it means the user can setup some window to autowrap, with others not autowrapping, if we put the hook in the buffer, the windows are forced to follow the same behaviour.

Its more flexible to put that in window scope.